packages/contracts/contracts/voting/token/TokenVotingSetup.sol

```diff
@@ -132,7 +130,7 @@ contract TokenVotingSetup is PluginSetup {
                 // it's simply ERC20 which gets checked by _isERC20
                 // Currently, not a satisfiable check..
```

Comment:
Tiny silly change that does nothing when you get it right, but makes things feel careless when it's not polished. 

```suggestion
                // Currently, not a satisfiable check.
```